README.md

```diff
@@ -698,6 +699,41 @@ The above command will result in exposing the node port 33080 for Webhook events
 external loadbalancer targeted to the node port, and register the hostname or the IP address of the external loadbalancer
 to the GitHub Webhook.
 
+##### Install with Kustomize
+
+To install this feature using Kustomize, add the resource to your `kustomization.yaml` file as in the example below:
+
+```yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- github.com/actions-runner-controller/actions-runner-controller/config//github-webhook-server?ref=v0.22.2
+```
```

Comment:
```suggestion
To install this feature using Kustomize, add `github-webhook-server` resources to your `kustomization.yaml` file as in the example below:

```yaml
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization

resources:
# You should already have this
- github.com/actions-runner-controller/actions-runner-controller/config//default?ref=v0.22.2
# Add the below!
- github.com/actions-runner-controller/actions-runner-controller/config//github-webhook-server?ref=v0.22.2
```
```

Just trying to make it extra clear by reflecting your pr description here 😄